tests/integ/test_diffusers.py

```diff
@@ -0,0 +1,84 @@
+import os
+import re
+from io import BytesIO
+
+import boto3
+from integ.utils import clean_up, timeout_and_delete_by_name
+from PIL import Image
+from sagemaker import Session
+from sagemaker.model import Model
+
+
+os.environ["AWS_DEFAULT_REGION"] = os.environ.get("AWS_DEFAULT_REGION", "us-east-1")
+SAGEMAKER_EXECUTION_ROLE = os.environ.get("SAGEMAKER_EXECUTION_ROLE", "sagemaker_execution_role")
+
+
+def get_framework_ecr_image(registry_id="763104351884", repository_name="huggingface-pytorch-inference", device="cpu"):
```

Comment:
Could you use the utilities out of sagemaker python sdk for this (https://sagemaker.readthedocs.io/en/stable/api/utility/image_uris.html), especially since you are already importing that library?

Reply:
Yes! but there is currently no way to have a container available since the dockerfile is not in this repo.